web/modules/custom/mymodule/mymodule.info.yml

```diff
@@ -0,0 +1,9 @@
+name: 'My Custom Module'
+type: module
+description: 'Creates a page showing "Sourav".'
+package: Custom
+core_version_requirement: ^10 || ^11
+
+dependencies:
+  - admin_toolbar
```

Comment:
Any specific reason why admin toolbar and admin toolbar tools are listed as dependencies here ?